llvm/lib/Target/AMDGPU/AMDGPU.h

```diff
@@ -92,7 +92,7 @@ class SILowerI1CopiesPass : public PassInfoMixin<SILowerI1CopiesPass> {
 
 void initializeAMDGPUDAGToDAGISelLegacyPass(PassRegistry &);
 
-void initializeAMDGPUAlwaysInlinePass(PassRegistry&);
+void initializeAMDGPUAlwaysInlinePass(PassRegistry &);
```

Comment:
Please don't reformat the whole file. You can use `git clang-format @^` to reformat only the parts you have modified. See https://clang.llvm.org/docs/ClangFormat.html#git-integration